### PR TITLE
add i18n support

### PR DESCRIPTION
**Pull request by kaidohallik**
fix hipster-labs/generator-jhipster-entity-audit#15

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -315,14 +316,32 @@ module.exports = yeoman.Base.extend({
...
+            );
+          }, this);
+        }
         this.copyFiles(files);
         // add bower dependency required
         jhipsterFunc.addBowerDependency('angular-object-diff', '1.0.3');
         jhipsterFunc.addAngularJsModule('ds.objectDiff');
         // add new menu entry
         jhipsterFunc.addElementToAdminMenu('entity-audit', 'list-alt', this.enableTranslation);
-        jhipsterFunc.addTranslationKeyToAllLanguages('entity-audit', 'Entity Audit', 'addAdminElementTranslationKey', this.enableTranslation);
```
Comment (deepu105) [inline generators/app/index.js]: why? this works the same way without needing a local file?

Comment (kaidohallik) [inline generators/app/index.js]: if global.json is missing in some language then yes, 'Entity Audit' is added to this language, rows 336-344 in this file

Comment (deepu105) [inline generators/app/index.js]: global.json should not be missing as it would break lot of translation then. Hence we dont have to handle it here

Comment (kaidohallik) [inline generators/app/index.js]: lines 336-344 reads `global.json` from template path, `global.json` contains exactly one key, for the admin menu, this PR adds translations for Estonian and English (`global.json` in template path), so initially only these 2 languages are supported and for the other languages text 'Entity Audit' is used

Comment (deepu105) [inline generators/app/index.js]: Ya but do we need to add that as its just one key and easy for users to change for their selected language

Comment (kaidohallik) [inline generators/app/index.js]: I tried to add complete i18n support. Do you see some drawback with this admin menu item translation implementation?

Comment (deepu105) [inline generators/app/index.js]: No drawback but I feel its unnecessary complication for the value it provides

Comment (kaidohallik) [inline generators/app/index.js]: Should I remove admin menu item i18n support from this PR?

**Comment by kaidohallik** [comment]
Conflict resolved, PR updated.

**Pull request merged**